airbyte-integrations/connectors/source-slack/metadata.yaml

```diff
@@ -6,7 +6,7 @@ data:
     hosts:
       - slack.com
   connectorBuildOptions:
-    baseImage: docker.io/airbyte/python-connector-base:4.0.0@sha256:d9894b6895923b379f3006fa251147806919c62b7d9021b5cd125bb67d7bbe22
+    baseImage: docker.io/airbyte/source-declarative-manifest:6.60.4@sha256:7a96257b67c4519dccbdf58535156a81efc5789936e71fb67ac399b6a1f6374a
```

Comment:
I assume `dockerImageTag` will change once the RC is merged

Reply:
yes, I'll update branch and bump version